portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      [" Your time is limited, so don't waste it living someone else's life. Don't be trapped by dogma, which is living with the results of other people's thinking. And most important, have the courage to follow your heart and intuition. They somehow already know what you truly want to become"];
```

Comment:
Do you want to add several greetings here, so that it's possible to see a different greeting each time the button is clicked?

Reply:
Yes, but I actually did it within the servlet section. I stored several  greetings in an array that randomly calls a greeting every time  the button gets pressed